src/components/heading/heading.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { TextStyle } from 'react-native';
+import { Text } from 'react-native-paper';
+
+import { HeadingLevel } from 'src/common/enums';
+import styles from './styles';
+
+type IHeadingProps = {
+  children: string;
+  level: HeadingLevel;
+  style?: TextStyle;
+};
+
+const Heading: React.FC<IHeadingProps> = ({ children, level, style }) => {
```

Comment:
We should set some default value for `level` here to avoid reaching `style[undefined]`